playwright/.auth/viewer.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Same with this file, probably an idea to gitignore the `playwright/.auth` folder entirely.

Reply:
Can we gitignore the whole `playwright` file just incase?